firebase-auth/src/commonMain/kotlin/dev/gitlive/firebase/auth/auth.kt

```diff
@@ -41,6 +41,15 @@ expect class AuthResult {
     val user: FirebaseUser?
 }
 
+expect class AuthTokenResult {
+    //    val authTimestamp: Long
+    val claims: Map<String, Any>
+    //    val expirationTimestamp: Long
+//    val issuedAtTimestamp: Long
+    val signInProvider: String?
+    val token: String?
```

Comment:
can they be null?

Reply:
Unfortunately the properties in Android SDK are marked as `@Nullable`, so didn't want to risk a crash.